components/Common/Card/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo, type ComponentProps, type FC } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+import { shortHumanReadableDate } from '@/util/shortHumanReadableDate';
+
+import styles from './index.module.css';
+
+type Author = {
+  firstName: string;
+  lastName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type CardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: ComponentProps<typeof Preview>['type'];
+  subtitle: string;
+  description: string;
+  author: Author;
```

Comment:
As I mentioned above, I think we can directly use the props of the `Avatar` component. Additionally, as stated in the issue description, it must be possible to have more than one avatar

```suggestion
  authors: ComponentProps<typeof Avatar>[];
```

Reply:
Thank you for your suggestion. As mentioned [here](https://github.com/nodejs/nodejs.org/pull/6037/files#r1367783451), using the `alt` prop directly conveys less meaning/intention than asking directly for a `firstName` and a `lastName`.